bats/tests/helpers/vm.bash

```diff
@@ -10,18 +10,18 @@ wait_for_shell() {
 }
 
 factory_reset() {
-    rdctl factory-reset
-
-    if is_unix; then
-        if pgrep -f rancher-desktop; then
-            pkill -f rancher-desktop
-            # Re-run because RD was not stopped before deleting the data
-            rdctl factory-reset
+    if using_npm_run_dev; then
+        if is_unix; then
+            run rdctl shutdown
```

Comment:
We probably want this regardless whether this is Unix.

Reply:
Probably, but we don't have anything implemented for Windows yet, so we can refactor once that is being implemented.